draft.py

```diff
@@ -258,4 +271,75 @@ def run_current_lot(self):
       yield time_remaining
     
     self.rotateCaptainList()
-    self.machine.nom_from_bid()
\ No newline at end of file
+    self.machine.nom_from_bid()
+
+  def checkNum(self, number):
+    if type(number) == float:
+      return True
+    return False
+  
+  def player(self, message):
+    message_parts = message.content.split()
+    player_name = message_parts[1]
+    if self.is_admin(message):
+      if self.checkPlayer(player_name):
+        raise AuctionValidationError(ClientMessage(
+          type=ClientMessageType.CHANNEL_MESSAGE,
+          data=f"Can't add {player_name}, that player is already in the system."
+          ))
+      if len(message_parts) > 2:
```

Comment:
instead of nesting ifs like this, it can be better to return early if the opposite is true, so like

```python
if not self.is_admin(message):
    return
<do more logic>
```

Reply:
Changed logic of player/captain methods